tools/cldr-apps/js/src/views/LockAccount.vue

```diff
@@ -0,0 +1,178 @@
+    } else if (user.userlevelName === "ADMIN") {
+      this.helpMessage = cldrText.get("lock_account_admin");
+    } else {
+      this.email = user.email;
+    }
+    this.summandA = Math.floor(Math.random() * 11);
+    this.summandB = Math.floor(Math.random() * 11);
+    this.sumCorrect = this.summandA + this.summandB;
+
+    this.cautionMessage = cldrText.get("lock_account_caution");
+    this.instructionMessage = cldrText.get("lock_account_instruction");
+
+    this.sumLabel = cldrText.sub("lock_account_sum", [
+      this.summandA,
+      this.summandB,
+    ]);
+    this.emailLabel = cldrText.get("lock_account_email");
+    this.reasonLabel = cldrText.get("lock_account_reason");
+    this.buttonLabel = cldrText.get("lock_account_button");
+  },
+
+  methods: {
+    lockAccount() {
+      this.errMessage = null;
+      if (this.sumCorrect !== parseInt(this.sumEntered)) {
+        this.errMessage = cldrText.get("lock_account_err_math");
+      } else if (this.email !== this.emailEntered) {
+        this.errMessage = cldrText.get("lock_account_err_email");
+      } else if (
+        !this.reasonEntered ||
+        !(this.reasonEntered = this.reasonEntered.trim())
+      ) {
+        this.errMessage = cldrText.get("lock_account_err_reason");
+      } else {
+        this.reallyLockAccount();
+      }
+    },
+
+    reallyLockAccount() {
+      this.loading = true;
+      this.errMessage = null;
+      const lockPostData = {
+        email: this.emailEntered,
+        reason: this.reasonEntered,
+        session: cldrStatus.getSessionId(),
+      };
+      const xhrArgs = {
+        url: cldrAjax.makeApiUrl("auth/lock", null),
+        postData: lockPostData,
+        handleAs: "json",
+        load: (json) => this.loadResult(json),
+        error: (err) => this.handleError(err),
+      };
+      cldrAjax.sendXhr(xhrArgs);
+    },
+
+    handleError(err) {
+      this.loading = false;
+      this.errMessage = err;
+    },
+
+    loadResult(json) {
+      this.loading = false;
+      this.finished = true;
+      const successMessage = cldrText.get("lock_account_success");
+      this.helpMessage = successMessage;
+      cldrStatus.setIsDisconnected(true);
+      cldrStatus.setSurveyUser(null);
+      cldrStatus.setPermissions(null);
+      cldrStatus.setSessionId(null);
+      // The success message will be displayed in the main window, replacing the form,
+      // as soon as Vue renders it. However, it may quickly be replaced with something else
+      // as a result of the disconnection, since the user has been logged out on the back end.
+      // To ensure the user has a chance to read the message, also open an alert box with the
+      // same message, then redirect to the default page. Ideally (though not necessarily) the
+      // alert should open after Vue has done the rendering, and before other messages start
+      // showing up about being disconnected, for which ten milliseconds seems to work OK.
```

Comment:
an alternative would be to fetch `POST /api/auth/login` with a payload of `{}` which will give a new anonymous session ID, that could be setup as the continuing session ID. It would be the effect of a refresh.

Reply:
it will return:
```json
{   "sessionId": "MTczMjg4NzIwMzEzNzgyMTc0NQo",  "user": false }
```

Reply:
That does sound better. I'll merge this PR as-is and then experiment with your idea...